packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -115,12 +115,17 @@ const ProductDetail = ({category, product, isLoading}) => {
     }
     const handleAddToCart = async (variant, quantity) => {
         try {
-            if (!variant?.orderable || !quantity) return
+            // @@@ Skip this check as it isn't applicable to standard products.
+            // if (!variant?.orderable || !quantity) { 
+            //     return
+            // }
+
             // The basket accepts an array of `ProductItems`, so lets create a single
             // item array to add to the basket.
             const productItems = [
                 {
-                    productId: variant.productId,
+                    // @@@ Standard products have an id, not a productId
+                    productId: variant.productId || variant.id,
```

Comment:
Variants and standard products have different attributes. We'll need to either update this method to allow it to handle both types of products or make separate methods to support the different product types.